Dockerfile

```diff
@@ -1,22 +1,18 @@
 # Mozilla Kinto server
-
-FROM node:lts-bullseye-slim as node-builder
-RUN apt-get update && apt-get install -y --no-install-recommends ca-certificates curl
-COPY scripts/build-kinto-admin.sh .
-COPY /kinto/plugins/admin ./kinto/plugins/admin
-RUN bash build-kinto-admin.sh
-
 FROM python:3.10-slim-bullseye as python-builder
 RUN python -m venv /opt/venv
-RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev
+RUN apt-get update && apt-get install -y --no-install-recommends build-essential libpq-dev ca-certificates curl
 ARG KINTO_VERSION=1
 ENV SETUPTOOLS_SCM_PRETEND_VERSION_FOR_KINTO=${KINTO_VERSION} \
     PATH="/opt/venv/bin:$PATH"
 # At this stage we only fetch and build all dependencies.
 WORKDIR /pkg-kinto
-COPY constraints.txt .
-COPY pyproject.toml .
+COPY constraints.txt pyproject.toml ./
 COPY kinto/ kinto/
+
+COPY scripts/pull-kinto-admin.sh .
+RUN bash pull-kinto-admin.sh
```

Comment:
It'd be nice to find a way to do this before we `COPY kinto/ kinto/`. Right now, any time we modify a Kinto source file, we'll have to redownload `kinto-admin`. We really only need to redownload Kinto Admin when `VERSION` changes.

Maybe we can pull kinto admin an earlier layer, then copy the files to the right place after we copy the `kinto` directory

Reply:
Made a change to do that, but it's a little janky because it's working around a circular dependency.

Reply:
Okay, you can leave it for now then -- since we're just curling, it's much faster that having to do a whole build step.